nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ExtractEmailHeaders.java

```diff
@@ -103,6 +102,21 @@
             .defaultValue("x-mailer")
             .build();
 
+    private static final String STRICT_ADDRESSING_DEFAULT_VALUE = "true";
+    public static final PropertyDescriptor STRICT_ADDRESSING = new PropertyDescriptor.Builder()
+            .name("STRICT_ADDRESS_PARSING")
+            .displayName("Use Strict Address Parsing")
+            .description("If true, strict address format parsing rules are applied to mailbox and mailbox list fields, " +
+                    "such as \"to\" and \"from\" headers, and FlowFiles with poorly formed addresses will be routed " +
+                    "to the failure relationship, similar to messages that fail RFC compliant format validation. " +
+                    "If false, the processor will extract the contents of mailbox list headers as comma-separated " +
+                    "values without attempting to parse each value as well-formed Internet mailbox addresses. " +
+                    "This is optional and defaults to " + STRICT_ADDRESSING_DEFAULT_VALUE)
+            .required(false)
+            .defaultValue(STRICT_ADDRESSING_DEFAULT_VALUE)
+            .allowableValues("true", "false")
```

Comment:
I would change this to use the "AllowableValue" class and move some of the description of "true" vs. "false" behavior from the description field into each value. Here is an example: https://github.com/apache/nifi/pull/2148/files#diff-4f399a5e493f124a256ae2740c20f839

Reply:
Sorry forgot about that. I had the "office plague" (flu) last week. I'll commit again very soon, and you can re-review.

Reply:
No worries! Will do, thanks!